packages/dd-trace/src/supported-configurations.json

```
@@ -161,6 +161,7 @@
     "DD_RUNTIME_METRICS_GC_ENABLED": ["A"],
     "DD_RUNTIME_METRICS_FLUSH_INTERVAL": ["A"],
     "DD_RUNTIME_METRICS_RUNTIME_ID_ENABLED": ["A"],
+    "DD_SERVERLESS_PUBSUB_ENABLED": ["A"],
```

```suggestion
    "DD_TRACE_SERVERLESS_PUBSUB_ENABLED": ["A"],
```

This aligns with other plugin names. I think we should use that.

---

updated to DD_TRACE_GCP_PUBSUB_PUSH_ENABLED